### ISSUE TITLE
Show Tables;

### ISSUE THREAD

**Issue by Hermann12** [dbcli/mycli#850]
Why can't the tool not use SHOW Tables; for MySQL database? Can I add this function in my personal environment?

**Comment by amjith** [comment]
I'm not sure what version of mycli you're using but `show tables` is fully supported. 

![image](https://user-images.githubusercontent.com/49260/79797056-72ac6100-830b-11ea-852a-709f06b6b48e.png)

What error message are you seeing when you run `show tables;` ?

**Comment by Hermann12** [comment]
I tried SHOW TABLES;  with capital letters like in MySQL She’ll. Let me try again, later.
